cookieConsent/CookieConsentFactory.js

```diff
@@ -0,0 +1,38 @@
+import * as OneTrust from "./OneTrust";
+
+const cookieConsent = {
+  OneTrust: OneTrust.default,
+};
+class CookieConsentFactory {
+  static initialize(sourceConfig, options) {
+    /**
+     *
+     * check which type of cookie consent manager needs to be called if enabled
+     * for now we have only OneTrust.
+     * But if new cookie consent manager options are implemented,
+     * we need to make sure only one of them is enabled by the user in the
+     * load options
+     *
+     */
+
+    if (
+      options.cookieConsentManager &&
+      options.cookieConsentManager.oneTrust &&
+      options.cookieConsentManager.oneTrust.enabled
```

Comment:
You can condense this as below,
```suggestion
     options.cookieConsentManager?.oneTrust?.enabled
```

Reply:
done